modules/FormView.js

```diff
@@ -382,6 +387,29 @@
     },
 
     /**
+     * @method  _generateSelectOptions
+     * @param element {Element} The select element to generate options for
+     * @param opts {Object} Additional behavior options for the bindings
+     * @param [opts.modelFormat] {Object} The function called before setting model values
+     * @private
+     * @return {<Stickit select options hash>}
+     */
+    __generateSelectOptions: function(element, opts) {
+      var collection = [],
+          options = $(element).children('option');
+
+      _.each(options, function(option) {
+        collection.push({'label': $(option).text(),
```

Comment:
last question. This all looks good, but I'm a little worried about blowing away your select options if they have different classes/attributes that you had added before. Will you lose classes?